packages/atlas/src/utils/number.ts

```diff
@@ -6,3 +8,21 @@ const numberFormatter = new Intl.NumberFormat('en-US', { maximumSignificantDigit
 export const formatNumber = (num: number): string => {
   return numberFormatter.format(num).replaceAll(',', ' ')
 }
+
+const TEN_BILLIONS = 10 ** 10
```

Comment:
Let's move this to `@/config/joystream` and call `HAPI_TO_JOY_RATE` or something along the lines